Jenkinsfile

```diff
@@ -0,0 +1,160 @@
+pipeline {
+  agent none
+  options {
+    disableConcurrentBuilds()
+  }
+  stages {
+    stage('restore-cache') {
+      agent {
+        label 'master'
+      }
+      steps {
+        sh """
+        if test -d '${env.JENKINS_HOME}/userContent/OMSimulator/testsuite/cache/${env.GIT_BRANCH}'; then
+          mkdir -p testsuite/cache/
+          cp -av '${env.JENKINS_HOME}/userContent/OMSimulator/testsuite/cache/${env.GIT_BRANCH}' testsuite/cache/
+        fi
+        """
+        stash name: 'old-linux64-db', includes: "testsuite/cache/${env.GIT_BRANCH}/linux64", allowEmpty: true
+        stash name: 'old-osx-db', includes: "testsuite/cache/${env.GIT_BRANCH}/linux64", allowEmpty: true
```

Comment:
Shouldn't this be "testsuite/cache/${env.GIT_BRANCH}/osx"?

Reply:
Yes, but there are more problems. I will be ready with the caching of the db-files soon